src/c_ndp.hpp

```diff
@@ -105,8 +105,8 @@ const std::array<unsigned char, 94> c_ndp::generate_neighbor_advertisement_new(
 		m_generate_neighbor_advertisement_packet.at(56) = 0x00;
 		m_generate_neighbor_advertisement_packet.at(57) = 0x00;
 
-		tab_view<uint8_t> source_destination_addr(m_generate_neighbor_advertisement_packet, 22, 32);
-		tab_view<uint8_t> header_with_content(m_generate_neighbor_advertisement_packet, 54, 40);
+		tab_view<uint8_t> source_destination_addr(&m_generate_neighbor_advertisement_packet.at(22), 32);
+		tab_view<uint8_t> header_with_content(&m_generate_neighbor_advertisement_packet.at(54), 40);
```

Comment:
Previous version was safer, as the constructor check is this e.g. 22 and 32 in range.